graphql/integration_test.go

```diff
@@ -56,6 +56,7 @@ func (s *graphQLSuite) SetupSuite() {
 	s.Require().NoError(testUser.Insert())
 	s.url = server.URL
 	s.apiKey = apiKey
+	s.apiUser = apiUser
```

Comment:
Doesn't look like this is used anywhere

Reply:
it's read here: https://github.com/evergreen-ci/gimlet/blob/master/middleware_auth_user.go#L185

Reply:
testuser was previously Unauthorized because authDataName was empty when it was passed to manager.GetUserByID